server/src/api/roster/roster.router.ts

```diff
@@ -54,6 +54,11 @@ router.get(
   controller.getRosterInfosForIndividual,
 );
 
+router.get(
+  '/info2/:edipi',
+  controller.getRosterInfosForIndividual,
+);
+
```

Comment:
This seems duplicate to `'/info/:edipi'` is this intentional?

Reply:
I think this is similar to my comment below...removing.